exonum-java-binding/service-archetype/src/main/resources-filtered/archetype-resources/__rootArtifactId__-service/pom.xml

```diff
@@ -0,0 +1,167 @@
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>\${groupId}</groupId>
+    <artifactId>\${rootArtifactId}-parent</artifactId>
+    <version>\${version}</version>
+  </parent>
+
+  <artifactId>\${rootArtifactId}-service</artifactId>
+  <version>\${version}</version>
+  <packaging>jar</packaging>
+
+  <name>Exonum Service</name>
+
+  <properties>
+    <nativeLibPath>\${env.EXONUM_HOME}/lib/native</nativeLibPath>
+    <!-- Service Artifact properties -->
+    <exonum.javaRuntimeId>1</exonum.javaRuntimeId>
+    <exonum.artifactName>\${project.groupId}/\${project.artifactId}</exonum.artifactName>
+    <exonum.artifactVersion>\${project.version}</exonum.artifactVersion>
+    <exonum.artifactId>
+      \${exonum.javaRuntimeId}:\${exonum.artifactName}:\${exonum.artifactVersion}
+    </exonum.artifactId>
+    <exonum.artifactFileName>
+      \${project.artifactId}-\${project.version}-artifact
+    </exonum.artifactFileName>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>\${groupId}</groupId>
+      <artifactId>\${rootArtifactId}-messages</artifactId>
+      <version>\${project.version}</version>
+    </dependency>
+
+     <!-- Exonum *must* have 'provided' scope so that it and its dependencies are not included
+         in the service artifact. Some of transitive dependencies of Exonum are safe
+         and recommended to use in a service in 'provided' scope as well, because they will not
+         be changed in an incompatible way in a compatible release of Exonum. Such dependencies
+         are listed in exonum-java-binding-bom (e.g., guava, guice, gson, vertx, pf4j,
+         protobuf-java). You can either use them implicitly as transitive dependencies
+         (no extra POM modifications needed) or define explicitly below with 'provided' scope
+         and NO version — it will be inherited from exonum-java-binding-bom.
+
+         See also https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Transitive_Dependencies
+         -->
+    <dependency>
+      <groupId>com.exonum.binding</groupId>
+      <artifactId>exonum-java-binding-core</artifactId>
+      <scope>provided</scope>
+    </dependency>
+
+    <!-- Override the scope of exonum-messages in case \${rootArtifactId}-messages depends on them -->
+    <dependency>
+      <groupId>com.exonum.messages</groupId>
+      <artifactId>exonum-messages</artifactId>
+      <scope>provided</scope>
+    </dependency>
+
+    <!-- Test dependencies -->
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.exonum.binding</groupId>
+      <artifactId>exonum-testkit</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
```

Comment:
I have doubts about test dependencies.
I do not see mockito usage in the archetype tests. So users can include it if needed.

Reply:
Yes, it is unused. It was originally added because we use Mockito widely (it is used in both services we have, QA and cryptocurrency, haven't check timestamping). But that's a good question how "helpful" with extra dependencies it shall get.